### ISSUE TITLE
Support semicolon syntax in go_package protobuf option

### ISSUE THREAD

**Issue by ivucica** [grpc-ecosystem/grpc-gateway#341]
Please support the following syntax to specify different package name:

```proto
option go_package = "example.com/pkg/project/proto;projectpb"
```

Current error:

```
E0330 11:14:26.373979   20973 generator.go:76] 11:15: expected declaration, found 'IDENT' projectpb:
// Code generated by protoc-gen-grpc-gateway                                                                     
// source: example.com/pkg/project/users.proto                                                                                                       
// DO NOT EDIT!                                                                                                                         
                                                                                   
/*                                                                                                                                                       
Package proto;authpb is a reverse proxy.                       
                                                                                                                                                                              
It translates gRPC into RESTful JSON APIs.                                     
*/                                                                                                                                                                     
package proto;projectpb                                                                                
import (
...
...
var (
...
)
--grpc-gateway_out: 11:15: expected declaration, found 'IDENT' projectpb                               
doc.go:7: running "protoc": exit status 1                                                                                   
```

See https://github.com/golang/protobuf/issues/139

**Comment by tmc** [comment]
Definitely support this.

**Comment by NeoCN** [comment]
Definitely support this! Expect just passing -grpc-gateway_out="$GOPATH/src" and then have  things Just Work.

**Comment by tmc** [comment]
@NeoCN thanks for your interest. Do you think you could take a crack at this?

**Comment by roadrunner** [comment]
official go_package syntax is the same way: `<FQ go package>;<short name>`
please support this.

**Comment by ivucica** [comment]
I cannot trivially reproduce this with current head. Has this been fixed? Can one of the other commenters repro this?

Tests do need to be updated to cover this, but I don't know if I'll have time to do that.

**Comment by roadrunner** [comment]
@ivucica i'm using 1.2.x for backward compatibility. no-chance to upgrade yet..

**Comment by rogerhub** [comment]
This should be fixed: #277 #462
